scripts/generate_mock_data.js

```diff
@@ -0,0 +1,40 @@
+const fetch = require("node-fetch");
+const mongoose = require("mongoose");
+
+const Resource = require("../backend/models/Resource");
+
+const JSON_LINK =
+  "http://www.json-generator.com/api/json/get/cecKjNYROW?indent=2";
+
+const createConnection = async () => {
+  await mongoose.connect(process.env.DB_URI, {
+    useUnifiedTopology: false,
+    useNewUrlParser: true
+  });
+};
+
+const closeConnection = async () => {
+  await mongoose.connection.close();
+};
+
+const fetchJson = async () => {
+  const response = await fetch(JSON_LINK);
+  const dataJSON = await response.json();
+  return dataJSON;
+};
+
+const addSampleResource = async resource => {
+  const newResource = new Resource(resource);
+  await newResource.save();
+};
+
+const main = async () => {
+  await createConnection();
+  const dataJSON = await fetchJson();
+  for (var i = 0; i < dataJSON.length; i++) {
```

Comment:
Looks good, Albert! 

Just a pro JS tip for you: whenever you are thinking of doing a `for` loop in JS, I suggest thinking: is it returning something? If so, can we use `map`? Is it just performing an action on each element? If so, `forEach` might work better.

Since `addSampleResource` is an `async` function, this means that it will return a Promise. Our goal is to wait for all the `.save()` requests to finish before moving on to `closeConnection()`. In order to do this, we can use `.map()` to map each of these database queries into a Promise object, and then use `await Promise.all([Array of promises])` which will complete only when all the promises passed in are complete.

See [here](https://github.com/hack4impact-uiuc/h4i-recruitment/blob/master/backend/src/api/futureInterview.js#L83) for an example of how I do this in recruitment portal.